wrapanapi/systems/redfish.py

```diff
@@ -33,5 +71,80 @@ def _identifying_attrs(self):
     def info(self):
         return 'RedfishSystem url={}'.format(self.url)
 
-    def __del__(self):
-        """Disconnect from the API when the object is deleted"""
+    def server_stats(self, *requested_stats, **kwargs):
+        # Get the requester which represents the class of this method's caller
```

Comment:
Docblocks explaining the intent of this and `server_inventory` methods (difference of counts, are they returning dicts, tuples, objects, names, etc.